File: decompiled/sources/com/hjq/http/config/RequestApi.java
package com.hjq.http.config;

import com.hjq.http.annotation.HttpIgnore;

public final class RequestApi implements IRequestApi {
    @HttpIgnore
    private final String mApi;

    public RequestApi(String str) {
        this.mApi = str;
    }

    public String getApi() {
        return this.mApi;
    }

    public String toString() {
        return this.mApi;
    }
}

File: decompiled/sources/com/hjq/http/config/RequestApi.kt
package com.hjq.http.config

import com.hjq.http.annotation.HttpIgnore

class RequestApi(str: String) : IRequestApi {
    @field:HttpIgnore
    private val mApi: String = str

    override fun getApi(): String {
        return mApi
    }

    override fun toString(): String {
        return mApi
    }
}
